mkdocs.yml

```diff
@@ -92,12 +53,47 @@ nav:
           - deployment/aws/index.md
           - Infrastructure: deployment/aws/infrastructure.md
           - CI/CD: deployment/aws/cicd.md
+
   - Contributing:
```

Comment:
And "Contribute" rather than "Contributing" :)

Reply:
agreed